stormpath/express-stormpath
# Add produces option for /logout

Implements the produces option for /logout.

##### How to verify

1. Checkout this branch.
2. Clone [express-stormpath-sample-project](https://github.com/stormpath/express-stormpath-sample-project).
3. Use `npm link` and `npm link express-stormpath` to link the library to the example project.
4. Start the server.
5. Navigate to `/login` and login as a user.
6. Navigate to `/logout`.
7. Verify that you're logged out.
8. Use either the Angular or React SDK.
9. Validate that you can logout (with a `application/json` request).

Fixes #341.
